gradle.properties

```diff
@@ -37,3 +37,6 @@ kotlin.mpp.androidSourceSetLayoutVersion=2
 
 # https://youtrack.jetbrains.com/issue/KT-42254
 kotlin.native.cacheKind=none
+
+# JDK Configuration
+org.gradle.java.home=/opt/homebrew/opt/openjdk@17/libexec/openjdk.jdk/Contents/Home
```

Comment:
you can put this into `local.properties` 

i have this set on my machine in `$HOME/.gradle/gradle.properties`